### PR TITLE
Replace Color palette (Primary & Overlay colors) and add intro description of HZA project to map.js

### LINKED ISSUE #67: [TASK] Replace commented out Intro Driver element
# Overview 🔎 
Currently the first step in the introductory driver/tour is commented out and needs updated to the HZA description. 

# Action 🏃 
Provided the copy below, replace the `element: #CtZoningAtlas` with `element: HiZoningAtlas` (_including the title and description_)

# The Copy
`{
      element: '#HiZoningAtlas',
      popover: {
        title: 'Hawaii Zoning Atlas',
        description: 'Zoning is the most important local law you've never heard of. Zoning defines where buildings can go, how large they can be, what they can be used for, and more. The current zoning laws prioritize single-family homes on large lots in much of the state, contributing to urban sprawl, traffic congestion, and rising housing costs. Our team read the complete zoning codes for all 4 counties and built this interactive map to show where housing can and can't be built across the state. We hope policymakers and housing advocates can use our data to make housing more affordable and equitable.',
        position: 'right'
      }
    }`

## Resources 👀 
[Hawaii-Zoning-Atlas](https://github.com/CodeforHawaii/Hawaii-Zoning-Atlas)/[scripts](https://github.com/CodeforHawaii/Hawaii-Zoning-Atlas/tree/main/scripts)/map.js
`lines 725-730`
Image

### LINKED ISSUE #71: Subtask: 🎨 Color palette (Primary & Overlay colors)
# Overview
Current base and overlay colors do not make for a pleasant or accessible viewing experience. A purposeful and tested color palette has been designed by UI/UX for dev implementation. 

# Action Item
Replace the maps current base and overlay colors with this palette (_see resources below_)

image

# Resources
[Figma](https://www.figma.com/file/T8pIgx0d4iOwb2CkxUoKn6/Web-Application?node-id=0%3A1&t=rSSgHneMJt7AYL5x-0) 🕺 💄

### PR DESCRIPTION
**Pull request by operator130**
Closes #71

### CODE REVIEW

File: scripts/map.js [hunk 1]
```diff
@@ -6,11 +6,11 @@ var dataLayer // GeoJSON layer with district data
 var overlays = {} // An object to contain overlay layer groups, eg `transit`
 
 var zone2color = {
-  R: '#645097', // primarily residential, satisfied
-  M: '#815196', // mixed with residential, satisfied
-  N: '#BA6CA4', // nonresidential, satisfied
-  NS: '#d0d0d0', // not satisfied
-}
+  R: "#BB22CA", // primarily residential, satisfied
+  M: "rgba(113, 70, 116, 0.92)", // mixed with residential, satisfied
```
Comment (kmal808) [inline scripts/map.js]:
```suggestion
  M: "#714674eb", // mixed with residential, satisfied
```
Tip: You can use 8 character Hex codes (like #11223344) for transparent RGBA colors

File: scripts/map.js [hunk 2]
```diff
@@ -6,11 +6,11 @@ var dataLayer // GeoJSON layer with district data
...
+  N: "rgba(113, 70, 116, 0.67)", // nonresidential, satisfied
```
Comment (kmal808) [inline scripts/map.js]:
```suggestion
  N: "#714674ab", // nonresidential, satisfied
```

File: scripts/map.js [hunk 3]
```diff
@@ -579,11 +579,11 @@ var loadDHHL = function () {
       weight: 1.5,
       spaceWeight: 1,
       angle: -45,
-      color: 'rgb(147, 94, 59)',
-      spaceColor: '#9cb4dc',
+      color: "rgba(250, 174, 123, 0.76)",
```
Comment (kmal808) [inline scripts/map.js]:
```suggestion
      color: "#faae7bc2",
```

**Review by kmal808 — CHANGES_REQUESTED** [review]
Nitpick changes, otherwise looks great 💯

**Comment by kmal808** [comment]
With all of the commits from #76 included in this PR, I think it makes sense to close 76 and merge them here. This will close #67 and #71. @operator130 do you mind editing the title of this to something that covers the combined changes made? Thanks 🤝

File: scripts/map.js (hunk 1 above)
Comment (operator130) [reply scripts/map.js]: Oh! I didn't know the longer hex codes altered the opacity! That's good to know!

**Comment by operator130** [comment]
Closes #67 and #71

**Pull request merged**